react/features/analytics/functions.js

```diff
@@ -1,10 +1,15 @@
 // @flow
 
-import JitsiMeetJS, { isAnalyticsEnabled } from '../base/lib-jitsi-meet';
+import JitsiMeetJS, {
+    analytics,
+    isAnalyticsEnabled
+} from '../base/lib-jitsi-meet';
 import { getJitsiMeetGlobalNS, loadScript } from '../base/util';
 
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
+export const sendEvent = analytics.sendEvent.bind(analytics);
```

Comment:
Again, do we know for sure that there will always be an analytics object?

Reply:
See the previous answer!